roles/os-cinder/README.md

```diff
@@ -1,42 +1,61 @@
-# Ansible role: OpenStack-Cinder
-
+# os-cinder
 This role setups the OpenStack Block Storage service
 
-# Requirements
-- A previously setup of the OpenStack Identity service
-where this role can create the users, roles, etc.
-
-- A previously setup messaging server like RabbitMQ
-
-# Role variables
-All variables of this role are defined in `defaults/main.yml`
-
-## Mandatory variables
-* `cinder_devices` *This is a list of devices*
-
-  ```
-    Example:
-
-    cinder_devices:
-      - /dev/sdb
-      - /dev/sdc
-      - /dev/sdd
-      - /dev/sdf
-  ```
-* `cinder_user_password`
-* `cinder_database_password`
-
-## Handlers
-* `restart cinder controller`
-* `restart cinder storage`
-
-# Dependencies
-This role needs the OpenStack-common (os-common) role.
-
-# Example playbook
-
-# License
+## Requirements
+- A previously setup database server.
+  You can setup one with `mariadb` role
+- A previously setup OpenStack Identity service.
+  You can setup one with `os-keystone` role
+- A previously setup messaging server.
+  You can setup one with `rabbitmq` role.
+
```

Comment:
[same comment](https://github.com/clearlinux/clear-config-management/pull/20/files#r70143074) applies here for previous 3 role lines (point to URL of the roles)